dynamic-endpoint-passthrough/service.bal

```diff
@@ -0,0 +1,11 @@
+import ballerina/http;
+
+configurable string invoke_url = ?;
+
+service / on new http:Listener(9090) {
+    resource function get greeting(string name) returns json|error {
+        http:Client clientEP = check new (invoke_url);
+
+        return check clientEP->/(name = name);
+    }
+}   
```

Comment:
Please check the newline here.

Reply:
Addressed in https://github.com/wso2/choreo-samples/pull/155/commits/dae18a703dca4bcef0a1f06be5d5875f22cfc1a3